Havtorn/Source/Core/EngineTypes.h

```diff
@@ -30,4 +31,67 @@ namespace Havtorn
 		Sequencer,
 		Script
 	};
-}
\ No newline at end of file
+
+	static SColor GetAssetTypeColor(const EAssetType type)
+	{
+		switch (type)
+		{
+		case EAssetType::Animation:
+			return SColor::Blue;
+		case EAssetType::Material:
+			return SColor::Green;
+		case EAssetType::SkeletalMesh:
+			return SColor::Magenta;
+		case EAssetType::StaticMesh:
+			return SColor::Teal;
+		case EAssetType::Texture:
+			return SColor::Red;
+		case EAssetType::Script:
+			return SColor::Blue;
+		default:
+			return SColor::White;
+		}
+	}
+
+	static std::string GetAssetTypeName(const EAssetType type)
+	{
+		switch (type)
+		{
+		case EAssetType::Animation:
+			return "Skeletal Animation";
+		case EAssetType::Material:
+			return "Material";
+		case EAssetType::SkeletalMesh:
+			return "Skeletal Mesh";
+		case EAssetType::StaticMesh:
+			return "Static Mesh";
+		case EAssetType::Texture:
+			return "Texture";
+		case EAssetType::Script:
+			return "Script";
+		default:
+			return "Asset";
+		}
```

Comment:
If we're happy with how magic enum formats these names this could be nice to avoid having to add any more here later down the line.
```suggestion		
		 return magic_enum::enum_name<EAssetType>(type).c_str();
```

Reply:
I liked the idea of having space spearated names for this function, but the convenience is worth it probably 👍